planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1722,6 +1737,7 @@ std::optional<LaneChangePath> NormalLaneChange::calcTerminalLaneChangePath(
   const auto lane_changing_time = PathShifter::calcShiftTimeFromJerk(
     shift_length, lane_change_parameters_->lane_changing_lateral_jerk, max_lateral_acc);
 
+  const auto & target_lane_length = common_data_ptr_->transient_data.target_lane_length;
```

Comment:
```suggestion
  const auto target_lane_length = common_data_ptr_->transient_data.target_lane_length;
```
trivial type, so reference is not needed.